crud.lua

```diff
@@ -69,6 +70,10 @@ crud.unflatten_rows = utils.unflatten_rows
 -- @function truncate
 crud.truncate = truncate.call
 
+-- @refer cut_rows.call
+-- @function cut_rows
+crud.cut_rows = cut_rows.call
```

Comment:
Maybe it's better to place this function directly in `utils`? In the same way as for `unflatten_rows `.